app/src/sharedTest/java/org/oppia/android/app/help/HelpActivityTest.kt

```diff
@@ -0,0 +1,146 @@
+import org.oppia.android.domain.classify.rules.numberwithunits.NumberWithUnitsRuleModule
+import org.oppia.android.domain.classify.rules.numericinput.NumericInputRuleModule
+import org.oppia.android.domain.classify.rules.ratioinput.RatioInputModule
+import org.oppia.android.domain.classify.rules.textinput.TextInputRuleModule
+import org.oppia.android.domain.onboarding.ExpirationMetaDataRetrieverModule
+import org.oppia.android.domain.oppialogger.LogStorageModule
+import org.oppia.android.domain.oppialogger.loguploader.LogUploadWorkerModule
+import org.oppia.android.domain.oppialogger.loguploader.WorkManagerConfigurationModule
+import org.oppia.android.domain.question.QuestionModule
+import org.oppia.android.domain.topic.PrimeTopicAssetsControllerModule
+import org.oppia.android.testing.RobolectricModule
+import org.oppia.android.testing.TestDispatcherModule
+import org.oppia.android.testing.TestLogReportingModule
+import org.oppia.android.testing.time.FakeOppiaClockModule
+import org.oppia.android.util.accessibility.AccessibilityTestModule
+import org.oppia.android.util.caching.testing.CachingTestModule
+import org.oppia.android.util.gcsresource.GcsResourceModule
+import org.oppia.android.util.logging.LoggerModule
+import org.oppia.android.util.logging.firebase.FirebaseLogUploaderModule
+import org.oppia.android.util.parser.GlideImageLoaderModule
+import org.oppia.android.util.parser.HtmlParserEntityTypeModule
+import org.oppia.android.util.parser.ImageParsingModule
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Tests for [HelpActivity]. */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(application = HelpActivityTest.TestApplication::class, qualifiers = "port-xxhdpi")
+class HelpActivityTest {
+
+  @get:Rule
+  val activityTestRule: ActivityTestRule<HelpActivity> = ActivityTestRule(
+    HelpActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+  )
+
+  @Inject
+  lateinit var context: Context
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  @Test
+  fun testHelpActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(createHelpActivityIntent())
+    val title = activityTestRule.activity.title
+
+    // Verify that the activity label is correct as a proxy to verify TalkBack will announce the
+    // correct string when it's read out.
+    assertThat(title).isEqualTo(context.getString(R.string.help_activity_label))
+  }
+
+  private fun setUpTestApplicationComponent() {
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+  }
+
+  private fun createHelpActivityIntent(): Intent {
+    return HelpActivity.createHelpActivityIntent(
+      ApplicationProvider.getApplicationContext(),
+      isFromNavigationDrawer = false,
+      profileId = null
```

Comment:
Instead of null pass this as `0`

Reply:
@sakshampruthi Please do not resolve comments on your own because that way the reviewer won't know what to follow-up on. Instead reply to the comment and based on which the reviewer can know if the comment should be resolved or not.

Reply:
Okay, extremely sorry, will keep that in mind from now

Reply:
No issues.